SECURITY.md

```diff
@@ -0,0 +1,24 @@
+# Security Policy
+
+## Supported Versions
+
+Security updates are applied to the latest MINOR version of Jekyll, and the version used by GitHub Pages, v3.9.x.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 4.2.x   | :white_check_mark: |
+| 3.9.x   | :white_check_mark: |
+| < 3.8.x | :x:                |
+
+## Reporting a Vulnerability
+
+Please report vulnerabilities by sending an email to security@jekyllrb.com with the following information:
+
+1. A description of the vulnerability
+1. Reproduction steps and/or a sample site (share a private repo to the Jekyll Core Team and @parkr)
+2. Your contact information
+
+The Jekyll security team will respond to your submission and notify you whether it has been confirmed by the team. 
+Your confidentiality is kindly requested as we work on a fix. We will provide our patch to you to test and verify that the vulnerability has been closed.
```

Comment:
What if the researcher provides a patch attached with the submission? How will the security team respond? How will they test?

> > How will they test?

*We could have a dedicated private repo for this, wired to GH Actions CI (chargeable beyond free minutes per month). The workflow clones given repository and the release tag / `*-stable` branch as needed..*

Reply:
For reference, GitHub offers a private fork and the ability to issue cve. But it doesn't do actions.